src/openrct2-ui/windows/TrackDesignPlace.cpp

```diff
@@ -628,9 +669,13 @@ static void window_track_place_draw_mini_preview_maze(
             {
                 uint8_t* pixel = draw_mini_preview_get_pixel_ptr(pixelPosition);
 
-                // Entrance or exit is a lighter colour
-                uint8_t colour = mazeElement->type == 8 || mazeElement->type == 128 ? PALETTE_INDEX_PRIMARY_LIGHTEST
-                                                                                    : PALETTE_INDEX_PRIMARY_MID_DARK;
+                uint8_t colour = _PaletteIndexColourTrack;
+
+                // Draw entrance and exit with different colours.
```

Comment:
Just say what the colours are.

Reply:
There are comments here https://github.com/OpenRCT2/OpenRCT2/pull/8491/files/63bad6dfa431a7e6219eede202c55e1c2c4d2a24#diff-b82e44e91487b8ee5305cdc3393109e4R36